packages/app/src/searchFilters.tsx

```diff
@@ -86,6 +86,11 @@ export const parseQuery = (
     const [key, values] = filter.condition.split(
       isExclude ? ' NOT IN ' : ' IN ',
     );
+
+    if (!key || !values) {
```

Comment:
I think using `filters` for deeplinking conditions but not parsing the filters here results in all of the deeplink filters being cleared when I add any other filter.

Reply:
good catch, I will keep this line as it breaks if someone passes in any other SQL.. but I added a check before this for BETWEEN/range syntax!